packages/playwright/src/cdp.ts

```diff
@@ -163,7 +157,7 @@ export const clickElement = async (page: Page, args: { id: string }) => {
   return true
 }
 
```

Comment:
Playwright no longer makes it easy to pluck out these Protocol types. This is still correctly typed